core/eolearn/tests/test_eodata.py

```diff
@@ -18,19 +18,20 @@
 from eolearn.core.eodata_io import FeatureIO
 from eolearn.core.exceptions import EODeprecationWarning
 from eolearn.core.types import FeatureSpec, FeaturesSpecification
-from eolearn.core.utils.testing import assert_feature_data_equal
+from eolearn.core.utils.testing import assert_feature_data_equal, generate_eopatch
 
 DUMMY_BBOX = BBox((0, 0, 1, 1), CRS(3857))
 
 
 @pytest.fixture(name="mini_eopatch")
 def mini_eopatch_fixture() -> EOPatch:
-    eop = EOPatch(bbox=DUMMY_BBOX)
-    eop.data["bands"] = np.arange(2 * 3 * 3 * 2).reshape(2, 3, 3, 2)
-    eop.data["zeros"] = np.zeros((2, 3, 3, 2), dtype=float)
-    eop.mask["ones"] = np.ones((2, 6, 6, 1), dtype=int)
-    eop.mask["twos"] = np.ones((2, 3, 3, 2), dtype=int) * 2
-    eop.mask_timeless["threes"] = np.ones((3, 3, 1), dtype=np.uint8) * 3
+    eop = generate_eopatch(
+        {
+            FeatureType.DATA: ["bands", "zeros"],
+            FeatureType.MASK: ["ones", "twos"],
+            FeatureType.MASK_TIMELESS: ["threes"],
+        }
+    )
```

Comment:
the names kinda dont make sense anymore :see_no_evil: but i think we can leave it as is, unless you have some good ideas.

Reply:
have the same problem maybe standard: CLP, mask, CLM, LULC ...

Reply:
can be "A", "B", "C" as well i guess.

Reply:
renamed